### PR TITLE
Support tar.gz file

### PR DESCRIPTION
**Pull request by msabramo**
so I don't need to create my own GitHub repo just so that I can get a `.zip` URL.

Extracted from #8353, which had a test failure in `harfbuzz` I didn't understand, so I'm attempting to isolate the cause of it.

### CODE REVIEW

**Comment by msabramo** [comment]
Probably want #8353 instead of this.

File: tools/system_libs.py [hunk 1]
```diff
@@ -872,7 +872,12 @@ def fetch_project(name, url, subdir, is_tarbz2=False):
           Ports.clear_project_build(name)
           return
 
-    fullpath = fullname + ('.tar.bz2' if is_tarbz2 else '.zip')
+    if is_tarbz2:
+        fullpath = fullname + '.tar.bz2'
```
Comment (kripken) [inline tools/system_libs.py]: as mentioned in the other PR, indentation should be 2 spaces on these lines

Comment (msabramo) [reply tools/system_libs.py]: OK, fixed the indentation.

Also to help prevent these kinds of things in the future, I submitted #8359.

**Comment by msabramo** [comment]
Screen Shot 2019-03-27 at 4 07 24 PM

🎉

**Comment by kripken** [comment]
Great, thanks! (We need you in AUTHORS, but I see that's in another PR, so we can land this.)

**Pull request merged**